CI.rst

```diff
@@ -145,7 +145,7 @@ have to be percent-encoded when you access them via UI (/ = %2F)
 +--------------+----------------------------------------------------------+----------------------------------------------------------+
 
 * <BRANCH> might be either "main" or "v2-*-test"
-* <X.Y> - Python version (Major + Minor).Should be one of ["3.7", "3.8", "3.9"].
+* <X.Y> - Python version (Major + Minor).Should be one of ["3.8", "3.9", "3.10"].
```

Comment:
Should this include 3.11 now?

Reply:
Yep. It's a draft i keep on rebasing and fixing basic issues incrementallly - so those things will have to be reviewed and updated

Reply:
solved.